content/docs/contributing/policy.md

```diff
@@ -3,59 +3,125 @@ title: Feature Policy
 description: 'cert-manager contributing guide: Contribution Policy'
 ---
 
-We are open to feature requests and PRs implementing those. If you plan on contributing a new feature into cert-manager we recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
+We love to receive both feature requests and PRs which add stuff to cert-manager; the community is at the heart of what we do!
```

Comment:
```suggestion
We love to receive both feature requests and PRs which improve to cert-manager; the community is at the heart of what we do!
```

I don't like the phrase "add stuff" (but also don't mind if it remains that much).

Reply:
Fair enough, I get it; will change :+1: